(In r24148) Closes scala/bug#4204, an issue with fsc.  When I switched to process
I mistakenly used a synchronous call when creating the server.
No review.
